src/Microsoft.DotNet.GitHub.Authentication/GitHubTokenProvider.cs

```diff
@@ -84,9 +84,8 @@ private bool TryGetCachedToken(long installationId, out AccessToken cachedToken)
 
         AccessToken token = _tokenCache[installationId];
 
-        // If the cached token will expire in less than 30 minutes we won't use it and let GetTokenForInstallationAsync generate a new one
         // and update the cache
```

Comment:
curious: why is part of this comment still here❓ and why not instead update the comment to say "15 minutes"❓

Reply:
Ah, I didn't see the other line.

It's technically not 15 minutes, it's 15 minutes before the token expires. So if the token is issued with 4 hour long longevity, it will technically last 3,75 hours. I thought it's misleading. Will remove the other line tomorrow.

Reply:
Oh and also it seems to have worked - the exceptions are down to zero:
![image](https://github.com/user-attachments/assets/95cbb569-a498-49c8-ab02-c698f982917a)